File: src/main/java/de/burrotinto/burroPlayer/values/PinValue.java
package de.burrotinto.burroPlayer.values;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

/**
 * Created by derduke on 20.02.17.
 */
@Setter
@Getter
@Component
@NoArgsConstructor
@AllArgsConstructor
@ConfigurationProperties(prefix = "burroplayer.adapter.PI4J.pins")
public class PinValue {
    private int happening;
    private int moviestatus;
}

File: src/main/kotlin/de/burrotinto/burroPlayer/values/PinValue.kt
package de.burrotinto.burroPlayer.values

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.stereotype.Component

/**
 * Created by Florian Klinger on 06.07.17, 22:05.
 */
@Component
@ConfigurationProperties(prefix = "burroplayer.adapter.PI4J.pins")
class PinValue(var happening: Int = 0, var moviestatus: Int = 0)